src/main/java/com/exasol/releasedroid/adapter/maven/MavenPlatformValidator.java

```diff
@@ -29,6 +30,25 @@ public Report validate() {
         final var report = ValidationReport.create();
         report.merge(validateRepositories(this.repository.getRepositoryValidators()));
         report.merge(validateFileExists(this.repository, MAVEN_WORKFLOW_PATH, "Workflow for a Maven release."));
+        report.merge(validatePom(this.repository.getMavenPom()));
```

Comment:
I think we should also add this to project-keeper. There we can also add it automated.